cypress/e2e/app_tests/test_3.cy.js

```diff
@@ -0,0 +1,10 @@
+describe('Form layout page', () => {
+    it('Type data in Horizontal form', () => {
+        cy.visit('/pages/forms/layouts');
+        cy.get('.form-horizontal .col-sm-9 .input-full-width.size-medium.shape-rectangle[type="email"]').clear().type('test@email.com')
+        cy.get('.form-horizontal .col-sm-9 .input-full-width.size-medium.shape-rectangle[type="password"]').clear().type('123456)')
+        
+        cy.get('.checkbox .label .custom-checkbox').click()
+        cy.get('.appearance-filled.size-medium.status-warning.shape-rectangle.transitions').click()
```

Comment:
I'd recommend to use here :
cy.contains('Sign in').eq(1)

Reply:
Here updated with in such way cy.get('.form-horizontal').contains('Sign in').eq(0).click(), because with an approach cy.contains('Sign in').eq(1), cypress throws assertion